rwatch/ui/notification_window.c

```diff
@@ -276,10 +280,10 @@ static void _up_single_click_handler(ClickRecognizerRef _, void *_w) {
 
 static void _notification_window_click_config_provider(NotificationWindow *w) {
     window_single_repeating_click_subscribe(BUTTON_ID_DOWN, 500, _down_single_click_handler);
-    window_single_repeating_click_subscribe(BUTTON_ID_UP  , 500, _up_single_click_handler  );
+    window_single_repeating_click_subscribe(BUTTON_ID_UP, 500, _up_single_click_handler);
```

Comment:
The extra spaces here were actually intentional, to make it visually easier to see the parallel between these lines :)